pkg/object/globalfilter/globalfilter.go

```diff
@@ -145,44 +157,49 @@ func (gf *GlobalFilter) Inherit(superSpec *supervisor.Spec, previousGeneration s
 	gf.reload(previousGeneration.(*GlobalFilter))
 }
 
-// BeforeHandle before handler logic for beforePipeline spec
-func (gf *GlobalFilter) BeforeHandle(ctx context.HTTPContext) {
-	handler, ok := gf.getPipeline(beforePipelineKey)
-	if !ok {
+func (gf *GlobalFilter) Handle(ctx context.HTTPContext, httpHandle protocol.HTTPHandler) {
```

Comment:
Please fix this Github action warning... Except this one, I think it looks great to me!

Reply:
Thanks, I dealt with this warning